### DISCUSSION TITLE
gouravkhunger/9-hand-picked-free-heroku-alternative-hosting-platforms

### DISCUSSION THREAD

**Discussion by giscus** [genicsblog/comments#130] [Announcements]
# gouravkhunger/9-hand-picked-free-heroku-alternative-hosting-platforms

Heroku has shut down their free tier and the free dynos began being deleted starting November 28, 2022. Here are top 9 alternatives to Heroku free plan to deploy your APIs and Websites seamlessly, without any restrictions!

https://genicsblog.com/gouravkhunger/9-hand-picked-free-heroku-alternative-hosting-platforms

**Comment by vincenzo** [comment]
You are missing a pretty important one: https://platform.sh  :)

**Reply by gouravkhunger** [reply]
Hey @vincenzo, thank you for the comment!

I did consider https://platform.sh but owing to the fact that it does not have a free plan, it did not make it to the list. This does not demean the service itself. But sorry, the list only contains truly free resources :)

**Comment by tessflightcontrol** [comment]
Any chance you'd add https://www.flightcontrol.dev/

Developer-first AWS Infrastructure designed for teams that want the benefits & reliability of AWS without hiring devops
